revolv/templates/project/project.html

```diff
@@ -17,53 +22,118 @@
+        <div class="overlay">
+            <div class="donate-button-header">
+               <a class="donate-link" href="#donate" data-reveal-id="donate-modal"><p>DONATE</p></a>
+            </div>
+            <p class ="project-title" >{{ project.title }}</p>
+        </div>
+    </div>
+    {% endif %}
+</div>
+
+
+<div class="row about">
+    <div class="header-text">
+        <p class="header">Promoting healthy business, healthy people, and a healthy planet</p>
+    </div>
+    <div class="gray-bar"></div>
+    <div class="row categories">
+        <div class="category-text"><p class="category-text-inside">CATEGORIES   </p></div>
+        {% for category in project.categories %}
+            <div class="category-holder">
+                <p class="category-label">{{ category }}</p>
+            </div>
+        {% endfor %}
+
+        <!-- These two are in here for testing and viewing purposes: projects don't have categories right now. -->
+        <div class="category-holder"><p class="category-label">ARTS</p></div>
+        <div class="category-holder"><p class="category-label">HEALTH</p></div>
+    </div>
+    {% if project.video_url %}
+        <iframe width="100%" height="500" type="text/html" class="project_video youtube-player" frameborder="0" allowfullscreen>
+        </iframe>
+    {% endif %}
+
+    <div class="about-text"><p>{{ project.org_about }}</p>
     </div>
 </div>
 
+{% if updates|length > 0 %}
+<div class="row project-updates">
+    <div class="header-text">
+        <p class="header">Project Updates</p>
+    </div>
+    <div class="gray-bar"></div>
+    {% for update in updates %}
+        <div class="update-container">
+            <div class="large-2 small-12 columns">
+                <div class = "date-container">
+                    <p class="date-year">{{ update.date.day }}</p>
+                    <p class="date-month-day">{{ update.month_word }} {{ update.year_word }}<p>
+                </div>
+            </div>
+            <div class="large-10 small-12 columns">
+                <p class="update-text">{{ update.update_text }}</p>
+            </div>
+        </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+<div class="row map">
+    <div id="map-canvas"></div>
+    <div class="overlay">
+        <div class="text-container">
+            <div class="title-container">
+                <p>{{ project.title }}</p>
+            </div>
+            <div class="address-container">
+                <p>{{ project.location_street }}</p>
+                <p>{{ project.location_city_state_zip }}</p>
+            </div>
+        </div>
+
+        <!-- this donate button doesnt work because the model has a top attribute of like 4000 pixels -->
+        <a href="#donate" data-reveal-id="donate-modal"><div class="donate-button-header">
+            <p>DONATE</p>
+        </div></a>
+    </div>
+</div>
+
+<div class="storage-div" id="{{ project.partial_completeness }}">
```

Comment:
I'm sure there's a non-hacky way to do this but I just did this so i could get the partial completeness in the javascript

how would i access it without the hackyness of this div?